textkit/install_dependencies.py

```diff
@@ -0,0 +1,50 @@
+import os
+import click
+import nltk
+from textkit.utils import read_tokens, output
+
+
+@click.command('install_dependencies')
+@click.option('--path', 
+              help='Separator between token and count in output.',
+              show_default=True)
```

Comment:
This is actually optional. I couldn't find a way to specify this! I saw `required=True` in the docs.... so not sure.